### PR TITLE
Update linux-3.2.35 to linux-3.2.69

### PR DESCRIPTION
**Pull request by cobracmder**
Update linux-3.2.35 to linux-3.2.69, the latest LTS release from the
3.2 branch for the upcoming 2015.08 ONIE release.

Platform testing (building) was performed on all platforms except
those setting their kernel explicitly:
accton_as5700_96x - fsl-sdk-v1.5
accton_as5710_54x - fsl-sdk-v1.5
accton_as6700_32x - fsl-sdk-v1.5
accton_as6710_32x - fsl-sdk-v1.5
accton_as7710_32x - fsl-sdk-v1.7
alphanetworks_snh60a0_320f - 3.14.27
centec_e580_48x6q - fsl-sdk-v1.5
centec_e580_48x2q4z - fsl-sdk-v1.5
centec_e580_20q4z - fsl-sdk-v1.5
fsl_p2020rdbpca - 4.1.1
kvm_x86_64 - 4.1.1
mlnx_x86 - 3.10.0

### CODE REVIEW

**Comment by cobracmder** [comment]
One thing to note on this PR, the patch for the s6000 now has a couple of warnings coming from it:
```
<stdin>:47: trailing whitespace.

<stdin>:103: trailing whitespace.

warning: 2 lines add whitespace errors.
done
```

I don't mind redoing the patch if I can figure out which lines they are referring to or if this is even an issue.

File: build-config/conf/kernel/3.2.69/linux.x86_64.config [hunk 1]
```diff
@@ -1507,7 +1510,7 @@ CONFIG_BCMA_POSSIBLE=y
 # CONFIG_EZX_PCAP is not set
 # CONFIG_MFD_CS5535 is not set
 # CONFIG_MFD_TIMBERDALE is not set
-# CONFIG_LPC_SCH is not set
+CONFIG_LPC_SCH=y
 # CONFIG_MFD_RDC321X is not set
```
Comment (cbrune) [inline build-config/conf/kernel/3.2.69/linux.x86_64.config]: These configs come from machine/dell/dell_s6000_s1220/kernel/platform-dell-s6000-s1220.patch 
```
CONFIG_I2C_ISCH
CONFIG_I2C_ISMT
CONFIG_GPIO_SCH
CONFIG_MFD_CORE
CONFIG_LPC_SCH
```

As such they should *not* be set in the generic x86 config.  They should remain commented out with the `# CONFIG_XXX is not set` text.

File: machine/dell/dell_s6000_s1220/kernel/platform-dell-s6000-s1220.patch [hunk 2]
```diff
@@ -1,4 +1,17 @@
-Platform support for the DELL S6000 with the Centerton S1220 CPU module
```
Comment (cbrune) [inline machine/dell/dell_s6000_s1220/kernel/platform-dell-s6000-s1220.patch]: I'm not sure why this patch had to change....

**Comment by cbrune** [comment]
The new kernel configs look good, modulo the comments I already made.

I'm not sure why the dell_s6000 platform patch needed to change.

**Comment by cobracmder** [comment]
I've updated the linux config to fixup the module selections.

The patch had to be updated because the main file (```arch/x86/kernel/reboot.c```) changed too much for the default patch to apply.  I didn't know how to update the patch in-situ so I recreated the patch manually followed by the ```git format-patch```.  If the unmodified patch can be applied to the new sources, I would like to learn how to do it and will update this PR again.

File: machine/dell/dell_s6000_s1220/kernel/platform-dell-s6000-s1220.patch [hunk 3]
```diff
@@ -37,12 +50,12 @@ index e61f79c..b1a63c7 100644
 +	}
 +	return 0;
 +}
-+
++
```
Comment (cbrune) [inline machine/dell/dell_s6000_s1220/kernel/platform-dell-s6000-s1220.patch]: white space here?

File: machine/dell/dell_s6000_s1220/kernel/platform-dell-s6000-s1220.patch [hunk 4]
```diff
@@ -93,7 +106,7 @@ index 7f43cf8..b1acf68 100644
 +	 Enable support for DELL S6000 switching platform.  This
 +	 platform has a S1220 Atom (Centerton) CPU module, a Broadcom
 +	 Trident2 switch ASIC and 32x40G QSFP ports.
-+
++
```
Comment (cbrune) [inline machine/dell/dell_s6000_s1220/kernel/platform-dell-s6000-s1220.patch]: white space

**Comment by cobracmder** [comment]
Updated dell s6000 patch so that it applies cleanly without whitespace warnings.

**Comment by cbrune** [comment]
Looks good.  Thanks!

Pushed to master:

- 38e2769 Update linux-3.2.35 to linux-3.2.69

**Pull request closed without merging**